src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -121,434 +121,439 @@ static ApplicationInsightsTelemetry()
-                        "Microsoft.Online.SharePoint.PowerShell",
-                        "shieldedvmdatafile",
-                        "shieldedvmprovisioning",
-                        "shieldedvmtemplate",
-                        "SkypeOnlineConnector",
-                        "SkypeForBusinessHybridHealth",
-                        "smbshare",
-                        "smbwitness",
-                        "smisconfig",
-                        "softwareinventorylogging",
-                        "SPFAdmin",
-                        "Microsoft.SharePoint.MigrationTool.PowerShell",
-                        "sqlps",
-                        "SqlServer",
-                        "StartLayout",
-                        "StartScreen",
-                        "Storage",
-                        "StorageDsc",
-                        "storageqos",
-                        "Storagereplica",
-                        "Storagespaces",
-                        "Syncshare",
-                        "System.Center.Service.Manager",
-                        "TLS",
-                        "TroubleshootingPack",
-                        "TrustedPlatformModule",
-                        "UEV",
-                        "UpdateServices",
-                        "UserAccessLogging",
-                        "vamt",
-                        "VirtualMachineManager",
-                        "vpnclient",
-                        "WasPSExt",
-                        "WDAC",
-                        "WDS",
-                        "WebAdministration",
-                        "WebAdministrationDsc",
-                        "WebApplicationProxy",
-                        "WebSites",
-                        "Whea",
-                        "WhiteboardAdmin",
-                        "WindowsDefender",
-                        "WindowsDefenderDsc",
-                        "WindowsDeveloperLicense",
-                        "WindowsDiagnosticData",
-                        "WindowsErrorReporting",
-                        "WindowServerRackup",
-                        "WindowsSearch",
-                        "WindowsServerBackup",
-                        "WindowsUpdate",
-                        "wsscmdlets",
-                        "wsssetup",
-                        "wsus",
-                        "xActiveDirectory",
-                        "xBitLocker",
-                        "xDefender",
-                        "xDhcpServer",
-                        "xDismFeature",
-                        "xDnsServer",
-                        "xHyper-V",
-                        "xHyper-VBackup",
-                        "xPSDesiredStateConfiguration",
-                        "xSmbShare",
-                        "xSqlPs",
-                        "xStorage",
-                        "xWebAdministration",
-                        "xWindowsUpdate",
-                    };
+                s_knownModules = new Lazy<HashSet<string>>(GetKnownModules);
```

Comment:
Can we could avoid the Lazy<T> allocation with `LazyInitializer.EnsureInitialized`, like:

```csharp
private static HashSet<string> KnownModules => LazyInitializer.EnsureInitialized(ref s_knownModules, GetKnownModules);
```